newrelic/alert/alert_channel.tf

```diff
@@ -25,6 +25,16 @@ resource "newrelic_alert_channel" "tfer--amos-002D-stone-002D-amos-002D-stone-00
   type = "user"
 }
 
+resource "newrelic_alert_channel" "tfer--hbrien-002D-newrelic-002D-com-002D-3370989" {
```

Comment:
Are the funky IDs for consistency of re-importing? My preference would be for them to be more meaningful, but could live with it.